pkg/juicefs/mount/builder/common.go

```diff
@@ -86,6 +87,13 @@ func (r *Builder) getVolumes() []corev1.Volume {
 				Path: config.JFSConfigPath,
 				Type: &dir,
 			},
+		}}, {
+		Name: "updatedb",
+		VolumeSource: corev1.VolumeSource{
+			HostPath: &corev1.HostPathVolumeSource{
+				Path: "/etc/updatedb.conf",
```

Comment:
If there is no /etc/updatedb.conf in host, the mount pod will fail, right?

Reply:
`volumeSource.Type` is set `HostPathFileOrCreate `. If there is no /etc/updatedb.conf in host, it will be created.